src/ui/layout/constraints.md

```diff
@@ -2003,16 +2052,16 @@ BoxConstraints.tight(Size size)
 ```
 
 If you revisit [Example 2](#example-2) above,
-it tells us that the screen forces the red
-`Container` to be exactly the same size as the screen.
-The screen does that, of course, by passing tight
+the screen forces the red `Container` to be
+exactly the same size as the screen.
+The screen achieves that, of course, by passing tight
 constraints to the `Container`.
 
-A _loose_ constraint, on the other hand,
-sets the **maximum** width and height, but lets the widget
-be as small as it wants. In other words,
-a loose constraint has a **minimum** width and height
-both equal to **zero**:
+Some boxes _loosen_ the constraints,
```

Comment:
i think there's a nuance lost in this edit. The concept of a "loose constraint" is distinct from the concept of "loosening" a constraint. A loose constraint is just one that has a minimum of zero and a maximum non-zero. It could be created from whole cloth, or it could be created based on an incoming constraint; only in the latter case is it considered a _loosening_.